Description: TemplateHaskell is unused
 Some modules enable the TemplateHaskell extension without actually using it.
Author: Joachim Breitner <nomeata@debian.org>
Forwarded: https://github.com/yesodweb/persistent/issues/427

--- haskell-persistent-2.1.6.orig/Database/Persist/Quasi.hs
+++ haskell-persistent-2.1.6/Database/Persist/Quasi.hs
@@ -2,7 +2,6 @@
 {-# LANGUAGE CPP #-}
 {-# LANGUAGE PatternGuards #-}
 {-# LANGUAGE ViewPatterns #-}
-{-# LANGUAGE TemplateHaskell #-}
 module Database.Persist.Quasi
     ( parse
     , PersistSettings (..)
--- haskell-persistent-2.1.6.orig/Database/Persist/Sql/Raw.hs
+++ haskell-persistent-2.1.6/Database/Persist/Sql/Raw.hs
@@ -1,4 +1,3 @@
-{-# LANGUAGE TemplateHaskell #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE FlexibleContexts #-}
 module Database.Persist.Sql.Raw where
